lib/explorer/polars_backend/series.ex

```diff
@@ -173,7 +173,7 @@ defmodule Explorer.PolarsBackend.Series do
   # Aggregation
 
   @impl true
-  def count(series), do: Shared.apply_series(series, :s_size)
+  def count(series), do: Shared.apply_series(series, :s_count)
```

Comment:
```suggestion
  def count(series), do: size(series) - nil_count(series)
```

Reply:
Yeah good thought! I'll apply this commit then remove the rust parts.